.tours/codeql-tutorial.tour

```diff
@@ -12,7 +12,8 @@
     },
     {
       "directory": "codeql-tutorial-database",
-      "description": "To run a CodeQL query, we need to select a CodeQL database.\n\nWe have prepared a sample database for you to use in this tutorial.\nLook for the `codeql-tutorial-database` directory highlighted in the Explorer view in your left sidebar.\n\nTo select this as your current database, right-click on the directory, and click `CodeQL: Set Current Database`. Click `Next` when you have done that."
+      "description": "To run a CodeQL query, we need to select a CodeQL database.\n\nWe have prepared a sample database for you to use in this tutorial, namely the `codeql-tutorial-database` directory.\n\nWe have selected this as your current database.",
+      "commands": ["codeQL.setDefaultTourDatabase"]
```

Comment:
So we have a hardcoded command in the extension that sets the tour database? Would it be possible to call the existing setCurrentDatabase command but pass it the specific URI of the tour DB? (I admit I tried this and CodeTour was not consistently working with URI arguments to commands, but it may be something we collaborate on with the CodeTour developers internally.)

Reply:
That's a good point. I was also keen to keep the config inside the codespace template instead of having it in the extension. However, even with having the URI segments deduced or storing a "preferred" DB, we'll still need a custom command in the extension to make sense of it since the `setCurrentDatabase` command expects a VSCode URI type.

Reply:
Actually scratch that, we can pass the URI just fine to `setCurrentDatabase`. I'll have another poke tomorrow to see if it's straightforward.

Reply:
@shati-patel and I looked into this for a bit. We can't see a clear way to trigger code from the `codeql-tutorial.tour` file. There doesn't seem to be an option in the schema for providing custom scripts: https://github.com/microsoft/codetour#tour-schema.

In terms of setting a "preferred" DB, we'd still need some sort of code to build the URI and load it into the tour step.  We're open to suggestions on this, but we're gonna set it aside for now!

Reply:
Agree that this is difficult to do from bash. The other option I've talked about before is to have the DB config JSON file declare which DB it considers selected. So when you load the workspace and its DB config, the correct DB is already selected. That only needs the path to the unzipped DB on the filesystem, not an internal URI. Is that viable?

Reply:
I spoke about this with @adityasharad and we decided that until the new database panel is available and made to work with local queries, this custom command is still our best option.